setup.py

```diff
@@ -119,6 +127,53 @@ def run(self):
 cmdclass['test'] = NoopTestCommand
 cmdclass['build_ext'] = BuildExtraLibraries
 
+
+def _download_jquery_to(dest):
+    # Note: When bumping the jquery-ui version, also update the versions in
+    # single_figure.html and all_figures.html.
+    url = "https://jqueryui.com/resources/download/jquery-ui-1.12.1.zip"
+    if not os.path.exists(os.path.join(dest, "jquery-ui-1.12.1")):
+        try:
```

Comment:
`os.makedirs(dest, exist_ok=True)`

Reply:
Written to be py2 compatible to simplify the backport (we can always update it later).